_posts/2015-11-15-sinatra-app.markdown

```diff
@@ -11,79 +11,166 @@ permalink: sinatra-app-guide
 ## What is Sinatra?
 <img src="/images/sinatrawiki.png">
 
-
 # Getting Started
 
-## *1.*Creating the application with Cloud9
+## *1.* Creating the application with Cloud9
 
 Follow the steps outlined in the Ruby Girls Cloud9 Sinatra Setup Guide [Cloud9 Setup](/cloud9-sinatra-guide)
 
-## *2.*Locate the terminal window in Cloud9
+## *2.* Write a description for your application
+
+Cloud9 automatically creates a file called `REAMDME.md`. The `.md` extension means that this is a MarkDown file.
+A readme file is useful for documenting what your application does, and how a developer can setup their environment to develop or run your application.
+There is no clearly defined Markdown standard, however, GitHub has their own [flavor](https://help.github.com/articles/github-flavored-markdown/)(sic).
+
+- [ ] Double click the `README.md` in the Cloud9 file explorer to open it.
+- [ ] Replace the contents of the file with your description.
+- [ ] Save the file.
+
+eg.
+```
+# My awesome Sinatra app!
+This app is like the RailsGirls Ideas app, but using Sinatra instead of Rails!
+```
+
+We will add more documentation as the project evolves.
+
+## *3.* Git
```

Comment:
I thought that git should be an initial step and that commits should keep happening throughout the tutorial. I asked a previous RailsGirls student and she agreed it's important to learn the correct processes for development and practice them. I got a bit slack towards the end of the tutorial with reminding to commit. It does seem a bit un-DRY....